src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -152,6 +153,60 @@ static DataTestUtility()
             }
         }
 
+        public static string SetupFileStreamDB()
+        {
+            try
+            {
+                if (!IsFileStreamDbSetup())
+                {
+                    if (!FileStreamDirectory.EndsWith("\\"))
+                    {
+                        FileStreamDirectory += "\\";
+                    }
+
+                    string dbName = GetUniqueName("FS", false);
+                    string createDBQuery = @$"CREATE DATABASE [{dbName}]
+                                         ON PRIMARY
+                                          (NAME = PhotoLibrary_data,
+                                           FILENAME = '{FileStreamDirectory}PhotoLibrary_data.mdf'),
+                                         FILEGROUP FileStreamGroup CONTAINS FILESTREAM
+                                          (NAME = PhotoLibrary_blobs,
+                                           FILENAME = '{FileStreamDirectory}Photos')
+                                         LOG ON
+                                          (NAME = PhotoLibrary_log,
+                                           FILENAME = '{FileStreamDirectory}PhotoLibrary_log.ldf')";
+                    using SqlConnection con = new(new SqlConnectionStringBuilder(TCPConnectionString) { InitialCatalog = "master" }.ConnectionString);
+                    con.Open();
+                    using SqlCommand cmd = con.CreateCommand();
+                    cmd.CommandText = createDBQuery;
+                    cmd.ExecuteNonQuery();
+                    _fileStreamDBName = dbName;
+                }
+            }
+            catch (SqlException e)
+            {
+                Console.WriteLine("File Stream database could not be setup. " + e.Message);
+                FileStreamDirectory = null;
+            }
+            return _fileStreamDBName;
+        }
+
+        internal static void DropFileStreamDb()
```

Comment:
Could you pass the required values as parameters here and move this function into the related test class?